web/common/AddressField.js

```diff
@@ -152,14 +169,29 @@ export function AddressField({
       onClose={() => {
         setOpen(false);
       }}
-      renderOption={(props, option) => (
-        <ListItemText
-          {...props}
-          primary={option.manual ? option.label : formatAddressMainText(option)}
-          primaryTypographyProps={{ className: "bold" }}
-          secondary={formatAddressSubText(option)}
-        />
-      )}
+      renderOption={(props, option) =>
+        option.activateLocation ? (
+          <Button
+            startIcon={<LocationOnIcon />}
+            variant="outlined"
+            className={classes.geolocationButton}
+            disableElevation
+            onClick={e => askCurrentPosition()}
+          >
+            Activer la géolocalisation
```

Comment:
I don't remember if we said we should change this to "Utiliser ma position actuelle" ?

Reply:
I'll ask Audrey tomorrow.

Reply:
You were right, I've changed the label and added the "information" panel